Rakefile

```diff
@@ -36,4 +36,4 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task :default => :test
+task default: [:test, :rubocop]
```

Comment:
The problem this patch try to resolve is: 

> Newcomers make a PR and got a error about rubocop only in the travis, how we can show that this project uses rubocop?

The proposed solution is

> Add the rubocop on the default rake task too, since this is natural to people run `$ rake` to check the suite and other things are ok

I think the autocorrect task is not related with this PR.
If the contributor want autocorrect him can use the optional param directly on the rubocop.

Make sense to you guys?